### DISCUSSION TITLE
Visium HD Quick start

### DISCUSSION THREAD

**Discussion by abaghian** [satijalab/seurat#8715] [Q&A]
For the visium HD tutorial (https://satijalab.org/seurat/articles/visiumhd_commands_intro) I will likely have to upgrade to Seurat v5 correct? 
I have downloaded the 8 and 16 um bins from 10x and created path to file with the directory containing the spatial folder, filtered features.h5 file, and others;  however, in contrast to visium v2  the Load10x_Spatial command here includes bin.size and my question is where is this data stored in the directory?
object <- Load10X_Spatial (data.dir = localdir, bin.size = c(8, 16))
Thank you!

**Comment by abaghian** [comment]
the actual question is what files are in your /outs folder

localdir <- "/brahms/lis/visium_hd/Visium_HD_Public_Data/HD_public_data/Visium_HD_Mouse_Small_Intestine/outs"

**Reply by Yijia-Jiang** [reply]
what should be the input file? Would specifying file name work for loading the data: Load10X_Spatial(data.dir = localdir, filename="filtered_feature_bc_matrix.h5") ?

**Reply by prasoon2012** [reply]
You don't need to mention the file name in case you use the bin.size option. It will read the necessary files.

**Comment by abaghian** [comment]
Figured it out. thank you
